ts/utils/strings.ts

```diff
@@ -66,3 +66,11 @@ export const formatTextRecipient = (e: EnteBeneficiario): string => {
 ${address}${civicNumber}\n
 ${cap}${city}${province}`.trim();
 };
+
+/**
+ * determine if the text is undefined or empty (or composed only by blanks)
+ * @param text
+ */
+export const isNullyOrEmpty = (text: string | undefined | null) => {
+  return (text || "").trim().length === 0;
```

Comment:
why don't to use `fromNullable` ?
something like: 
```javascript
fromNullable(text).fold(false, t => t.trim().length===0)
```
Just a suggestion, I think your solution works fine too!

Reply:
It is definitely better 
Update here https://github.com/pagopa/io-app/pull/1908/commits/b2647cedda917318f0805d7a0cf899f089dd886d